### PR TITLE
docs:Upgrade python-protobuf for docuploader

### PR DESCRIPTION
**Pull request by tmatsuo**
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp-spanner/634)

### CODE REVIEW

File: ci/kokoro/docker/publish-refdocs.sh [hunk 1]
```diff
@@ -65,6 +65,11 @@ echo "================================================================"
 echo "Installing docuploader package $(date)."
 python3 -m pip install --user gcp-docuploader
 
+
+echo "================================================================"
+echo "Upgrading protobuf package $(date)."
+python3 -m pip install --user protobuf --upgrade
```
Comment (coryan) [inline ci/kokoro/docker/publish-refdocs.sh]: Can we do both of these in a single pip command?

Comment (tmatsuo) [reply ci/kokoro/docker/publish-refdocs.sh]: Yes, done!

File: ci/kokoro/docker/build.sh [hunk 2]
```diff
@@ -341,6 +341,7 @@ fi
 
 if [[ "${BUILD_NAME}" == "publish-refdocs" ]]; then
   "${PROJECT_ROOT}/ci/kokoro/docker/publish-refdocs.sh"
+  exit_status=$?
```
Comment (coryan) [inline ci/kokoro/docker/build.sh]: FYI: this seems like an unrelated change, but I think I understand why we need it

Comment (tmatsuo) [reply ci/kokoro/docker/build.sh]: Yeah, with this the job will correctly report failures

**Pull request merged**